docs/flows-format.adoc

```diff
@@ -220,16 +227,22 @@ The "Cardinality" column gives information about the implied metric cardinality
 | fine
 | k8s.layer
 | `NetworkEvents`
-| string
-| Network events flow monitoring
+| object[]
+| Network events, such as network policy actions, composed of nested fields: +
+- Feature (such as "acl" for network policies) +
+- Type (such as an "AdminNetworkPolicy") +
+- Namespace (namespace where the event applies, if any) +
+- Name (name of the resource that triggered the event) +
+- Action (such as "allow" or "drop") +
+- Direction (Ingress or Egress)
```

Comment:
@jotak can we add a Note to this that Network Events is only available in 4.17 and 4.18 versions of OpenShift Container Platform?